section2-week1/exercise3/Main.cpp

```diff
@@ -0,0 +1,28 @@
+/*Exercise 3 - Positive, Negative, or Zero
+Write a program that will ask the user for a number and determine if it is a positive number, a negative number, or zero.
+*/
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int number;
+
+	cout << "do you want to prove my math skills? \n gimme a number and I say whether the number is a negative number, a positive number or cero " << endl;
+
+	cin >> number;
+
+	if (number < 0)
+	{
+		cout << "this number is negative " << endl;
+	}
+	if (number == 0)
```

Comment:
You're still checking if number is less than 0 AND if the number is 0, but these conditions could never be both true. You should instead use an `else if` here.